app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt

```diff
@@ -153,6 +153,20 @@ class ExternalAppBrowserFragment : BaseBrowserFragment(), UserInteractionHandler
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        requireComponents.analytics.metrics.track(
```

Comment:
Do we want to emit two separate events here or just track the time difference ourselves and emit that? Is nanos too fine, would millis work better? Implementation looks good otherwise.

Reply:
We haven't defined what time difference we care about yet, so this can be tweaked on Marissa's side without needing a code change.